### DISCUSSION TITLE
The 1.6 Release of Kade Engine

### DISCUSSION THREAD

**Discussion by Kade-github** [KadeArchive/Kade-Engine#1395] [General]
View the list of [changelogs here](https://kadedev.github.io/Kade-Engine/changelogs/changelog-1.6)

<hr /><em>This discussion was created from the release <a href='https://github.com/KadeDev/Kade-Engine/releases/tag/1.6'>The 1.6 Release of Kade Engine</a>.</em>

**Comment by BombasticTom** [comment]
let's fucking gooooooooooooooooooooooooooooooooooooooooooooooo

**Comment by Futuremappermydud** [comment]
Gaming

**Reply by c0deblox** [reply]
watching

**Comment by FinkoStinko** [comment]
Right when I'm making a mod too!! now i gotta switch n shit god damnn

**Reply by ACardboardBox1** [reply]
no, dont

**Reply by TowsterBusiness** [reply]
Please don't that's like a week of extra work

**Comment by DatRatVS** [comment]
![gimmy](https://user-images.githubusercontent.com/49768896/126574737-5ab68dad-f477-415f-ab43-6a6b1aeab166.png)

**Reply by c0deblox** [reply]
funi

**Comment by juney2008** [comment]
cool! also linux and mac build of 1.6 when?

**Reply by TowsterBusiness** [reply]
Already and thing lmao

**Reply by GechaTN1** [reply]
yea but when will they be an option on here instead of having to build it ourselves? it takes a long time

**Comment by NotShinx** [comment]
i have a problem, i'm making a FNF mod and now all the arrows are the down arrow, HELP

**Reply by TowsterBusiness** [reply]
Look into Note.hx and see what the prefixes are

**Reply by NotShinx** [reply]
> Look into Note.hx and see what the prefixes are

Tes i did ir but don't work. The prefixes are good like the KE 1.5 it works if i put the default note skin (with the skin prefix)

**Comment by Pandaity** [comment]
Somethings wrong I tried making it so all the weeks are all unlocked
![image](https://user-images.githubusercontent.com/85077779/126728975-18c158c7-0ddf-49b7-823a-107e266aa879.png)
but then this happened
![image](https://user-images.githubusercontent.com/85077779/126728982-3d381c2f-b9a2-499f-abc7-8704145f1227.png)

**Reply by karatejoe848** [reply]
hey try using this one
public static var weekUnlocked:Array<Bool> = [true, true, true, true, true, true, true, true, true, true, true, true, true, true];

**Comment by c0deblox** [comment]
I feel like there should be an update that makes it easy to mod

**Reply by c0deblox** [reply]
You know Roblox's Mic Drop key charting feature where you could chart a song in that game with your keyboard and the beat you put playing? I'd like to see that in Kade Engine.

Also there would be a feature if a key is being held for more than a certain amount of time it would register as a hold note and it the hold note would be longer depending on how long you hold the key

Another option would be if you hit a certain key, it would change from Player 1 to Player 2 or even both?
Heres some keybind ideas (could be changed if user wanted to):
Q: Player 1
W: Player 2
E: Both

Plus if you used the key charting thingy it would pull up the 3 2 1 go sound effect before you start

**Reply by DatRatVS** [reply]
You could code it in as your own Dev Tool.

**Comment by TheALt-creator** [comment]
there's no artifacts for windows  since  im on a windows computers

**Comment by MJFNF** [comment]
I know this is retarded but the charting option keep crashing on the web version

**Comment by theRealKingBanana** [comment]
can you make it so you can download the html5 version source code

**Comment by BishWithGun** [comment]
so um how do i get the file to work. like what do i have to do to get it to the right version.

**Comment by SuperSpinDashGamer39** [comment]
How do I chart songs in kade engine 1.6 its confusing

**Comment by ghost** [comment]
Should probably say that this update has charting completely broken for HTML5 use.
Pressing 7 just crashes the game.

**Comment by SuperSpinDashGamer39** [comment]
hey so how do I add new difficulties to kade engine I asked on several modding discords about it and I'm desprate
